rust-runtime/aws-smithy-async/src/future/rendezvous.rs

```diff
@@ -38,6 +37,35 @@ pub fn channel<T>() -> (Sender<T>, Receiver<T>) {
     )
 }
 
+/// Errors for rendezvous channel
+pub mod error {
+    use std::fmt;
+
+    /// Error when [crate::future::rendezvous::Sender] fails to send a value to the associated `Receiver`
+    #[derive(Debug)]
+    pub struct SendError<T> {
+        source: tokio::sync::mpsc::error::SendError<T>,
```

Comment:
Nit: import things when not in codegen =)

Reply:
Used an alias to bind the import to `TokioSendError` in 59ffe31.